### ISSUE TITLE
Contrast ratio of spinnaker.io body text possibly too low

### ISSUE THREAD

**Issue by ghost** [spinnaker/spinnaker.github.io#610]
TLDR: I'd like to propose that we move to a thicker font on spinnaker.io in order to increase the text contrast.

Main body text on spinnaker.io is quite thin, resulting in a lack of contrast between it and the page's white background.  Here is a screenshot that demonstrates how text looks on the site currently and how it appears with a thicker font:

![skinny_vs_thick_contrast_ratio](https://user-images.githubusercontent.com/34253460/37178537-ca38549c-22f0-11e8-8b5b-cc86649b2e53.png)

The difference in contrast is quite clear.  I've personally experienced a bit of eye strain while reading the site and I believe that the contrast ratio is quite possibly below minimum accessibility guidelines of 4.5:1.  It's difficult to test this accurately because the actual text color value is above accessibility guidelines but the thinness of the font reduces it.

This is an issue I've worked around on my local machine using a stylesheet override but I'd like to modify the site to use a thicker font or darker colour by default.
